core/chains/evm/monitor/balance_test.go

```diff
@@ -254,3 +256,24 @@ func Test_ApproximateFloat64(t *testing.T) {
 		})
 	}
 }
+
+func callbackOrTimeout(t testing.TB, msg string, callback func(), durationParams ...time.Duration) {
```

Comment:
Using variadic params for something optional is an anti-pattern. Let's find another way to do this.

Reply:
Oh I guess it was just moved? Not blocking then I guess, but I still think it's worth fixing - at least to say `timeout` instead of `duration` which doesn't tell you anything.

Reply:
This is the only place we call this function and durationParams is never used. I'll remove it instead.